### DISCUSSION TITLE
From Nextcloud AIO back to Nextcloud

### DISCUSSION THREAD

**Discussion by ralfrupf1976** [nextcloud/all-in-one#5027] [Questions] [answered]
I'm trying to convert my nextcloud aio instance back into a standard installation. In my experiment, standard means - based on Debian, Docker and Portainer.

This works so far except for importing the users. The cloud admin is imported, but all other users are not, for reasons I don't understand.

**What I did:**
- Stop all Containers of Nextcloud AIO
- Copy database-dump.sql from /var/lib/docker/volumes/nextcloud_aio_database_dump/_data
- Adjust user in database: sed -i 's/oc_nextcloud/oc_admin/g' database-dump.sql 
- Copy database to running nextcloud portainer (new instance) - docker cp database-dump.sql <ContainerID>:/database-dump.sql
- Login in to Database Container - docker exec -it <ContainerID> /bin/bash

**Delete & recreate Database, import dump:**
PGPASSWORD="xxxxxxxxxxxxxxxxxxxxxxx" psql -h localhost -U oc_admin -c "DROP DATABASE \"nextcloud_database\";" -d postgres

PGPASSWORD="xxxxxxxxxxxxxxxxxxxxxxxx" psql -h localhost -U oc_admin -c "CREATE DATABASE \"nextcloud_database\";" -d postgres
Import Dump

PGPASSWORD="xxxxxxxxxxxxxxxxxxxxxxxxxxxx" psql -h localhost -U oc_admin -d nextcloud_database -f database-dump.sql

**Restart Containers Nextcloud & Database**

_Login with admin on new nextcloud possible - but all users missing:
The number of existing users is displayed but the users themselves are missing_

**Comment by ralfrupf1976** [comment]
[docker compose.txt](https://github.com/user-attachments/files/16343941/docker.compose.txt)

**Comment by szaimen** [comment]
> I'm trying to convert my nextcloud aio instance back into a standard installation.

Hi, why do you want to do so?

**Reply by ralfrupf1976** [reply]
In my case - I need more users. Nextcloud AIO is great piece of software, easy to handle, working great. But limited to 100 users. I need actually around 120 users. Nextcloud offered 150 users - for a very high price. So I want to host nextcloud again with the complete free version.
By the way - perhaps there are other users who wants to flip back for other reasons.

**Comment by szaimen** [comment, marked as answer]
I see. I guess it would make most sense to open a new thread then at https://help.nextcloud.com as there are more users around there that can help

**Reply by ralfrupf1976** [reply]
Thanks and:

https://help.nextcloud.com/t/from-nextcloud-aio-back-to-nextcloud/198750
